go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/riywo/loginshell v0.0.0-20200815045211-7d26008be1ab // indirect
 	github.com/saracen/go7z-fixtures v0.0.0-20190623165746-aa6b8fba1d2f // indirect
 	github.com/saracen/solidblock v0.0.0-20190426153529-45df20abab6f // indirect
+	github.com/square/exit v1.3.0 // indirect
```

Comment:
Add dependencies on this because to use the predefined [exit codes](https://github.com/square/exit/blob/main/exit.go).

Reply:
If you're pulling this in, we probably won't need to define our own error type, as this package [has one](https://github.com/square/exit/blob/main/exit.go#L138) already.

Reply:
Great point. I removed our own type in 7c849c3eee591a08bde7567d24cc1471dfbfeca6 and it simplifies the code quiet a lot. Thank you